build.proj

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is this file needed?

Reply:
I have removed it to simplify it, it isn't necessary. However, the file is the official way how to make "dotnet build" to build using fake, see:  https://github.com/FakeBuild/fake-bootstrap/tree/paket_clitool

Reply:
Yeah, but we have also other bootstraps there (the batch and shell script). I don't see a reason to have a bootstrap commited if we do not expect it to be used. Or, do we use it?

Reply:
The other bootstraps were using it. However, as I wrote, I got rid of it. The only benefit was that .NET Core developers could be used to using dotnet build command. But they should be able to run a batch file instead.